### PR TITLE
feat: autofocus components on page change

### PR DESCRIPTION
**Pull request by SimonDmz**
configure orchestrator to allow autofocus on almost every component
version gap to prevent confusion with lunatic-v1 older versions

### CODE REVIEW

File: src/components/checkbox/checkbox-group/lunatic-checkbox-group.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import React from 'react';
 import CheckboxGroup from './html/checkbox-group';
 import LunaticComponent from '../../commons/components/lunatic-component-without-label';
+import React from 'react';
```
Comment (MickaelMenet) [inline src/components/checkbox/checkbox-group/lunatic-checkbox-group.js:3]: the react from "react" not located in the first place disturbs me.

File: src/components/checkbox/checkbox-one/html/checkbox-one.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import React from 'react';
+import './checkbox-one.scss';
+
 import PropTypes from 'prop-types';
 import RadioGroup from '../../../radio/html/radio-group';
+import React from 'react';
```
Comment (MickaelMenet) [inline src/components/checkbox/checkbox-one/html/checkbox-one.js:5]: Same here

File: src/components/checkbox/checkbox-one/lunatic-checkbox-one.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
-import React from 'react';
+import CheckboxOne from './html/checkbox-one';
 import LunaticComponent from '../../commons/components/lunatic-component-without-label';
+import React from 'react';
```
Comment (MickaelMenet) [inline src/components/checkbox/checkbox-one/lunatic-checkbox-one.js:3]: Same

**Comment by SimonDmz** [comment]
Refactor needed => pass autofocus and autofocusFn in Lunatic-context and remove propsDrilling

**Comment by SimonDmz** [comment]
replaced by https://github.com/InseeFr/Lunatic/pull/470

**Pull request closed without merging**